SU2_CFD/src/solvers/CNEMOEulerSolver.cpp

```diff
@@ -567,7 +593,9 @@ void CNEMOEulerSolver::SetTime_Step(CGeometry *geometry, CSolver **solver_contai
         Lambda_1 = (4.0/3.0)*(Mean_LaminarVisc+Mean_EddyVisc);
         Lambda_2 = (Mean_ThermalCond+Mean_ThermalCond_ve)/cv;
         Lambda   = (Lambda_1 + Lambda_2)*Area*Area/Mean_Density;
-        nodes->AddMax_Lambda_Visc(iPoint,Lambda);
+
+      if(nodes->GetSymmetry(iPoint) > 0 && config->GetMarker_All_KindBC(iMarker) == SYMMETRY_PLANE) nodes->AddMax_Lambda_Visc(iPoint, Lambda/2.0);
+      else nodes->AddMax_Lambda_Visc(iPoint, Lambda);
```

Comment:
I might be missing something because I am only looking at the changed lines, but, if you set the Normal to zero on symmetry planes then Lambda is 0 and all this logic is redundant, right?